backend/codesirius/problems/models/execution_constraint.py

```
@@ -0,0 +1,80 @@
+    language = models.ForeignKey(
+        Language, on_delete=models.CASCADE, related_name="execution_constraints"
+    )
+
+    class Meta:
+        verbose_name = "Execution Constraints"
+        verbose_name_plural = "Execution Constraints"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["problem", "language"], name="unique_problem_language"
+            )
+        ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)